source/blog/2015-07-15-submission-stats.markdown

```diff
@@ -14,52 +12,55 @@ We had 81 total submissions in 2015!
 
 <em><small>The above numbers are total submissions; about 14-18 submissions were tagged in 2 categories.</small></em>
 
-If workshops are 3 hours, talks 1, discussions 1, and lightning talks .25, that’s a grand total of **83 hours of content**! And while we seriously debated making SassConf two weeks long, in the end, we couldn’t pick everything. (Sad trombone.)
+If workshops are 3 hours, talks 1 hour, discussions 1 hour, and lightning talks 15 minutes, that's a grand total of **83 hours of content**! We  seriously debated making SassConf two weeks long, in the end we couldn’t pick everything. (Sad trombone.)
```

Comment:
@misscs @elyseholladay Maybe this gif? ![http://media.giphy.com/media/DKUiRQrNt9LWg/giphy.gif](too much content)